tests/dp_engine_test.py

```diff
@@ -161,8 +161,217 @@ def test_contribution_bounding_cross_partition_bounding_applied(self):
                     max_partitions_contributed, dict_of_pid_to_pk)))
 
     def test_aggregate_none(self):
-        self.assertIsNone(
-            pipeline_dp.DPEngine(None, None).aggregate(None, None, None))
+        with self.assertRaises(Exception):
+            pipeline_dp.DPEngine(None, None).aggregate(None, None, None)
+
+    def test_check_aggregate_params(self):
+        test_cases = [
+            {
+                "desc":
+                    "None col",
+                "col":
+                    None,
+                "params":
+                    pipeline_dp.AggregateParams(
+                        noise_kind=pipeline_dp.NoiseKind.GAUSSIAN,
+                        max_partitions_contributed=1,
+                        max_contributions_per_partition=1,
+                        metrics=[pipeline_dp.Metrics.PRIVACY_ID_COUNT]),
+                "data_extractor":
+                    pipeline_dp.DataExtractors(
```

Comment:
It looks like data extractors in most cases the same.
Nit: create variable before `test_cases` 

` default_extractors = ... # like now` and to use it

And maybe the same with `params` as well

It would be easier to see which parts of the tests are non-default